ames_qsar/out_fileread.py

```diff
@@ -0,0 +1,15 @@
+import os
+import glob
+
+def out_fileread(filepath):
+    os.chdir(filepath)
+
+    namelist = []
+    #This could be adjusted later to locate other extentions
+    for filename in glob.glob('*.out'):
+        
+        #removing the filepath leaves just the filename
+        filename = filename.replace(str(filepath), '')
+        namelist.append(filename)
+
+    return namelist

```

Comment:
This whole function could be done clearly in one line [make sure to look everything up!]:
```
return [basename(f) for f in glob.glob(os.path.join(filepath, '*.out'))]
```

Reply:
basename() performs differently in python than in Linux; it only gives you the directory name, not the files in it, at least as far as I can tell. I was able to simplify the function down to one line and use only the os functions. Tests pass